qa/tasks/cephfs/mount.py

```diff
@@ -142,6 +142,11 @@ def create_files(self):
                 'sudo', 'touch', os.path.join(self.mountpoint, suffix)
             ])
 
+    def test_create_file(self, filename='testfile', dirname=None, user=None,
+                         pwd=None, check_status=True):
+        return self.create_file(filename=filename, dirname=dirname, user=user,
+                         pwd=pwd, check_status=False)
+
```

Comment:
The subject on this patch should probably be "add a method to test if file can be created". This doesn't actually write to the file, AFAICT.

Reply:
You are right.